src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private double width;
+    private double height;
+
+    public Rectangle(String color ,double width,double height) {
+        super(color);
+        this.height= height;
+        this.width = width;
+    }
+
+    @Override
+    public void draw() {
+        super.draw();
```

Comment:
Ensure that the `super.draw()` call is necessary. This depends on the implementation of the `Figure` class. If `Figure` does not have a `draw()` method or if it does not need to be called, this line can be removed.